app/src/main/java/org/mifos/mobile/utils/UiState.kt

```diff
@@ -0,0 +1,7 @@
+package org.mifos.mobile.utils
+
+sealed class UiState {
```

Comment:
It is better to give a specific name like `RegistrationUiState` than a general name

Reply:
done, 😃